src/colorizer/TimeControls.ts

```diff
@@ -1,26 +1,27 @@
 import { DEFAULT_PLAYBACK_FPS } from "../constants";
 
-import CanvasWithOverlay from "./CanvasWithOverlay";
-
```

Comment:
Removed dependency on CanvasWithOverlay by passing in a `getFrameFn` on construction